crypto/ec/ec_lib.c

```diff
@@ -23,6 +23,7 @@
 
 /* functions for EC_GROUP objects */
 
+#ifndef OPENSSL_NO_DEPRECATED_3_0
 EC_GROUP *EC_GROUP_new_ex(OPENSSL_CTX *libctx, const EC_METHOD *meth)
 {
```

Comment:
It does not make sense to make a new 3.0 function immediately deprecated. Should it be added at all?

Reply:
Good point!

@bbrumley I believe this will also affect your documentation edits!